src/frontend/src/pages/ChangeRequestDetailPage/ImplementedChangesList.tsx

```diff
@@ -4,44 +4,45 @@
  */
 
 import { ImplementedChange } from 'shared';
-import { datePipe, emDashPipe, fullNamePipe, wbsPipe } from '../../utils/pipes';
+import { datePipe, fullNamePipe, wbsPipe } from '../../utils/pipes';
 import { routes } from '../../utils/routes';
 import { Link, ListItem, List, Typography } from '@mui/material';
-import PageBlock from '../../layouts/PageBlock';
 import { Link as RouterLink } from 'react-router-dom';
 import DynamicTooltip from '../../components/DynamicTooltip';
+import InfoBlock from '../../components/InfoBlock';
 
 interface ImplementedChangesListProps {
   changes: ImplementedChange[];
   overallDateImplemented?: Date;
 }
 
-const ImplementedChangesList: React.FC<ImplementedChangesListProps> = ({ changes, overallDateImplemented }) => {
```

Comment:
we should display overallDateImplemented on this page still, maybe put it next to the title: Implemented Changes - overallDateImplemented